tracking_manager/models/ir_model.py

```diff
@@ -0,0 +1,205 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from ast import literal_eval
+
+from odoo import api, fields, models, tools
+
+
+class IrModel(models.Model):
+    _inherit = "ir.model"
+
+    active_custom_tracking = fields.Boolean(default=False)
+    tracked_field_count = fields.Integer(compute="_compute_tracked_field_count")
+    automatic_custom_tracking = fields.Boolean(
+        compute="_compute_automatic_custom_tracking",
+        readonly=False,
+        store=True,
+        default=False,
+        help=("If tick new field will be automatically tracked " "if the domain match"),
```

Comment:
```suggestion
        help=("If tick new field will be automatically tracked if the domain match"),
```

Reply:
Thanks for your review @damdam-s , I applied your suggestions, except the last one.